README.md

```diff
@@ -1 +1,188 @@
-# Hello-Microverse-Project-Two
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+
+<div align="center">
+  
+
+  <h3><b>Hello Microverse Project Two</b></h3>
+
+</div>
+
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+
+# 📖 [Hello Microverse Project Two] <a name="about-project"></a>
+
+
+**[Hello Microverse Project Two]** is a project on use of linters that only focuses on two languages which are html/css at the moment but could be modified later.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html/css.org/">Html/css</a></li>
+  </ul>
+</details>
+
+
+
+
+
+### Key Features <a name="key-features"></a>
+
+- **[Added index.html]**
+- **[Better UI/UX Style]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Coming soon)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1. Web browser
+
+2. Code editor(visual studio code)
+
+3. Git/Github(version control system)
+
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+1.git clone <the ssh url of the given repository>
+
+2.cd <the name of the repository>
+
```

Comment:
- [OPTIONAL] I will strongly advise you to remove they open < and closed with > then add the appropriate url so that the user can use it directly.